File: desktop/src/com/gmaniliapp/flappyhermes/desktop/DesktopLauncher.java
package com.gmaniliapp.flappyhermes.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.gmaniliapp.flappyhermes.FlappyHermes;

public class DesktopLauncher {
	public static void main (String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		config.width = FlappyHermes.WIDTH;
		config.height = FlappyHermes.HEIGHT;
		config.title = FlappyHermes.TITLE;
		new LwjglApplication(new FlappyHermes(), config);
	}
}

File: desktop/src/com/gmaniliapp/flappyhermes/desktop/DesktopLauncher.kt
package com.gmaniliapp.flappyhermes.desktop

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration
import com.gmaniliapp.flappyhermes.FlappyHermes

object DesktopLauncher {
    @JvmStatic
    fun main(arg: Array<String>) {
        val config = LwjglApplicationConfiguration()
        config.title = FlappyHermes.TITLE
        config.width = FlappyHermes.WIDTH
        config.height = FlappyHermes.HEIGHT
        LwjglApplication(FlappyHermes(), config)
    }
}